scripts/build/build_libpq.sh

```diff
@@ -70,7 +109,55 @@ if [ "$ID" == "centos" ]; then
 fi
 
 
-if [ "$ID" == "centos" ]; then
+if [ "$ID" == "macos" ]; then
+
+    # Build kerberos if needed
+    krb5_dir="krb5-${krb5_version}/src"
+    if [ ! -d "${krb5_dir}" ]; then
+        "$curl" -sL \
+            curl -sL "https://kerberos.org/dist/krb5/$(echo 1.21.3 | grep -oE '\d+\.\d+')/krb5-${krb5_version}.tar.gz" \
```

Comment:
There are two errors here: curl is repeated twice, the version number stripping doesn't use the `krb5_version`. Note that you can strip a suffix using:
```
$ echo ${krb5_version%.*}
1.21
```

Reply:
I'm guessing you've resolved this one because you're mopping it up yourself along with your `brew uninstall curl` change?

Reply:
Yes, fixed